examples/create-env-cluster-topics/outputs.tf

```diff
@@ -31,15 +31,18 @@ output "kafka_cluster_id" {
   value = module.honest_labs_kafka_cluster_basic.kafka_cluster_id
 }
 
-output "kafka_topic_name" {
+output "kafka_topic_name_1" {
```

Comment:
missing descriptions on all the outputs.

Reply:
added for all